moveit_core/exceptions/src/exceptions.cpp

```diff
@@ -35,14 +35,17 @@
 /* Author: Acorn Pooley, Ioan Sucan */
 
 #include <moveit/exceptions/exceptions.h>
-#include <ros/console.h>
+#include <rcutils/logging_macros.h>
+#include "rclcpp/rclcpp.hpp"
+
+#define ROS_ERROR RCUTILS_LOG_ERROR
 
 moveit::ConstructException::ConstructException(const std::string& what_arg) : std::runtime_error(what_arg)
 {
-  ROS_ERROR_NAMED("exceptions", "Error during construction of object: %s\nException thrown.", what_arg.c_str());
+  ROS_ERROR("Error during construction of object: %s\nException thrown.", what_arg.c_str());
```

Comment:
this is a regression from moveit1 - we want to keep the namespaces

Reply:
Logging changed https://github.com/ros-planning/moveit2/pull/7/commits/7138f303bd5900532f0fabd9d19933013d164a0e